internal/cmp/constants.go

```diff
@@ -24,5 +24,19 @@ const (
 	tier1GatewayType             = "NSX-T Tier-1 Gateway"
 	routerFirewallExternalPolicy = "GatewayPolicy"
 
+	// load balancer consts
+	ApplicationProfile       = "application-profile"
+	ServiceTypeLBHttpProfile = "LBHttpProfile"
+	tier1                    = "/infra/tier-1s/26cdb82e-0057-4461-ad4d-cddd61d77b1f"
+	networkServerID          = 1
+	size                     = "SMALL"
```

Comment:
Please, if possible can we have a prefix of `default` for the `size` and `loglevel` , so that it is more readable.